tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_SqlExample.ipynb

```diff
@@ -556,6 +564,75 @@
    "source": [
     "results.success"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6914725d-85cc-4c9d-a93d-2d1c329eac74",
+   "metadata": {},
+   "source": [
+    "# Loading Data into Postgresql Database"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f4a183e-f0df-40f4-b64f-439666e32ab4",
+   "metadata": {},
+   "source": [
+    "* The following code can be used to build the postgres database used in this notebook. It is included (and commented out) for reference.\n",
+    "* In order to load the data into a local `postgresql` database, please feel free to use the `docker-compose.yml` file available at `great_expectations/assets/docker/postgresql/`. \n",
+    "\n",
+    "### To spin up the `postgresql` database\n",
+    "* Have [Docker Desktop](https://www.docker.com/products/docker-desktop/) running locally.\n",
+    "* Navigate to `great_expectations/assets/docker/postgresql/`\n",
+    "* Type `docker-compose up`\n",
+    "* Then uncomment and run the following snippet"
```

Comment:
Also added a section that describes how the data can be loaded so the notebook can be run locally (using Docker Desktop)